Source/OxyPlot/Series/FinancialSeries/CandleStickAndVolumeSeries.cs

```diff
@@ -101,7 +106,25 @@ public LinearAxis BarAxis
         /// <summary>
         /// Gets or sets the style of volume rendering
         /// </summary>
-        public VolumeStyle VolumeStyle { get; set; }
+        public VolumeStyle VolumeStyle 
+        { 
+            get
+            {
+                return this.volumeStyle;
+            }
+
+            set
+            {
+                // if style has changed adjust axis min/max
+                if (value != this.volumeStyle && this.VolumeAxis != null)
```

Comment:
other parts of the library put this responsibility on the client. Should be consistent + keep it simple